src/main/java/net/minecraftforge/fml/client/ClientModLoader.java

```diff
@@ -138,7 +139,12 @@ private static void finishModLoading(Executor executor)
 
     public static VersionChecker.Status checkForUpdates()
     {
-        return VersionChecker.Status.UP_TO_DATE;
+
+        boolean anyOutdated = ModList.get().getMods().stream()
+                .map(VersionChecker::getResult)
+                .map(result -> result.status)
+                .anyMatch(status -> status == VersionChecker.Status.OUTDATED || status == VersionChecker.Status.BETA_OUTDATED);
+        return anyOutdated ? VersionChecker.Status.OUTDATED : null;
```

Comment:
What's the reason behind chaning the default from `VersionChecker.Status.UP_TO_DATE` to `null`

Reply:
Because `UP_TO_DATE` is not necessarily right. If some mods are `AHEAD` or `BETA` then we certainly aren't up to date.
The reason for choosing `null` here is that that's what previous versions did.